sktime/utils/dependencies/_dependencies.py

```diff
@@ -48,6 +48,14 @@ def _check_soft_dependencies(
 
         is the same as ``_check_soft_dependencies(("package1", "package2"))``
 
+        If the import name differs from the package name used for installation
```

Comment:
I would put this explanation directly after the first block of explanations ending in `"pandas>1.2.3"`.

Also, I suggest you do not refer to "import namd" or "package name" since these are not terms that were discussed above and would introduce confusion, but start like "Note that the package name is what one writes in `pip install`, e.g., " or similar.